src/mfpbench/pd1/benchmark.py

```diff
@@ -201,7 +201,8 @@ def _results_for(
         features = xs.columns
         for metric, surrogate in self.surrogates.items():
             # We clip as sometimes the surrogate produces negative values
-            xs[metric] = surrogate.predict(xs[features]).clip(min=0)
+            bounds = self.Result.metric_defs[metric].bounds
+            xs[metric] = surrogate.predict(xs[features]).clip(min=0, max=(bounds[1]))
```

Comment:
I guess this could just be `.clip(*bounds)`?

Reply:
Changed that in the next commit